release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -390,6 +390,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-12-installer-bug-fixes"]
 ==== Installer
 
+Previously, the number of supported user defined tags was 8 and reserved {product-title} tags was 2 for AWS resources. With this release, the number of supported user defined tags is now 25 and reserved {product-title} tags is 25 for AWS resources. You can now add up to 25 user tags during installation. (link:https://issues.redhat.com/browse/CFE-622[*CFE#622*])
```

Comment:
You will need a bullet in front of the entry otherwise this lgtm! 

```suggestion
* Previously, the number of supported user defined tags was 8 and reserved {product-title} tags was 2 for AWS resources. With this release, the number of supported user defined tags is now 25 and reserved {product-title} tags is 25 for AWS resources. You can now add up to 25 user tags during installation. (link:https://issues.redhat.com/browse/CFE-622[*CFE#622*])
```